domain-services/ifix-es-pipeline/mvnw

```diff
@@ -0,0 +1,316 @@
+#!/bin/sh
```

Comment:
Unnecessary file. Please delete it.

Reply:
minor change, to be taken up as part of code cleanup.